en/docs/guides/organization-login/try-organization-login.md

```diff
@@ -67,6 +67,14 @@ To configure the business applications:
 
 7. Click **Update** to save the configurations.
 
+8.  Go to the **User Attributes** tab, and click on + Add User Attributes.
+    - Select <i>Email, First Name, Last Name</i> and <i>Username</i> from the list.
```

Comment:
Shall we add this as a separate step? step 9?

Reply:
Sure, I will update the document.

Reply:
Updated the PR.